### PR TITLE
Russian Language Support

### PR DESCRIPTION
**Pull request by Nekonyx**
Phrases translated: 986 out of 994 [99%]

Files:
[100%] [140/140] layout.php
[100%] [125/125] beatmaps.php
[100%] [118/118] forum.php
[92%] [92/100] users.php
[100%] [58/58] beatmapsets.php
[100%] [49/49] authorization.php
[100%] [42/42] community.php
[100%] [33/33] multiplayer.php
[100%] [31/31] common.php
[100%] [31/31] accounts.php
[100%] [25/25] contest.php
[100%] [21/21] status_page.php
[100%] [21/21] model_validation.php
[100%] [20/20] home.php
[100%] [19/19] admin.php
[100%] [19/19] password_reset.php
[100%] [18/18] store.php
[100%] [17/17] rankings.php
[100%] [16/16] user_verification.php
[100%] [14/14] events.php
[100%] [14/14] bbcode.php
[100%] [12/12] news.php
[100%] [11/11] artist.php
[100%] [11/11] errors.php
[100%] [8/8] wiki.php
[100%] [6/6] changelog.php
[100%] [5/5] beatmap_discussions.php
[100%] [4/4] supporter_tag.php
[100%] [3/3] livestreams.php
[100%] [2/2] validation.php
[100%] [1/1] api.php

Demo: [nneko.ru](http://nneko.ru)

### CODE REVIEW

**Comment by peppy** [comment]
Shall we merge what you've done so far, or should we hold off until it's 100%?

File: resources/lang/ru/home.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    'user' => [
+        'title' => 'новости',
+        'news' => [
+            'title' => 'Новости',
+            'error' => 'Не удалось загрузить последние новости, пробовали обновить страницу?',
+        ],
+        'header' => [
+            'welcome' => 'Привет, <strong>:username</strong>!',
+            'messages' => 'У тебя одно новое сообщение|У тебя :count новых сообщении',
```
Comment (EVAST9919) [inline resources/lang/ru/home.php]: `У тебя :count новых сообщении`
сообщениЙ

Comment (Nekonyx) [inline resources/lang/ru/home.php]: Да, в локалке у меня уже всё есть

**Comment by Nekonyx** [comment]
No, I need to push some changes.

**Comment by Nekonyx** [comment]
Need review.

**Comment by Nekonyx** [comment]
Ready to be merged :tada:

**Pull request merged**